kitchen-tests/.kitchen.travis.yml

```diff
@@ -2,7 +2,6 @@
 driver:
   name: dokken
   privileged: true
-  chef_image: chef/chef
```

Comment:
what was this there for?  beware that i had to do some hacky shit to make dokken work with chef_github provisioner, and applying standardized dokken config to this file may cause issues...